src/frontend/js/components/Search/index.tsx

```diff
@@ -114,6 +120,9 @@ const Search = ({ context }: CommonDataProps) => {
         )}
       </div>
       <div className="search__results">
+        <h2 className="offscreen">
```

Comment:
Actually I would like to question the idea of a hidden h2. Isn't it probematic for SEO?

Reply:
I must say I'm not sure if search engines conclude that it's shaddy behavior or not if they detect a hidden _heading_.

That being said, this h2 in particular has no real impact on the SEO. If it's ignored by search engines, it's ignoring a "search results" text.

One thing that some people do is use aria attributes to have the same accessibility meaning as heading tags, hopefully being ignored by search engines ("hopefully" because I actually never seen actual research on "are aria attributes understood by google or not"):

```diff
- <h2>test</h2>
+ <div role="heading" aria-level="2">test</div>
```

Reply:
@jbpenrath WDYT? I remember you handcrafted the header levels on the search page

Reply:
It's fine for me to use a `h2` here. From my side, I replaced the `p` element by a `h3` to display course title within CourseGlimpse components so about semantic hierarchy all is fine.

Reply:
ok then :+1: